data/docs/messaging-queues/confluent-kafka.mdx

```diff
@@ -0,0 +1,681 @@
+     -Dotel.instrumentation.kafka.experimental-span-attributes=true \
+     -Dotel.instrumentation.kafka.metric-reporter.enabled=true \
+     -jar /path/to/your/consumer.jar
+```
+
+### Step 4: OpenTelemetry Collector Setup
+
+Set up the OpenTelemetry Collector to collect JMX metrics from Kafka and spans from producer and consumer clients, then forward them to SigNoz.
+
+4.1 **Download the JMX Metrics Collector**
+
+Download the JMX metrics collector necessary for Kafka metrics collection. 
+
+You can download the latest `.jar file` for the opentelemetry jmx metrics using 
+this [release link](https://github.com/open-telemetry/opentelemetry-java-contrib/releases).
+
+4.2 **Install the OpenTelemetry Collector**
+
+Install the local OpenTelemetry Collector Contrib.
+
+
+**Using the Binary:**
+
+Download the OpenTelemetry Collector Contrib binary. You can [follow this doc](https://signoz.io/docs/tutorial/opentelemetry-binary-usage-in-virtual-machine/) to 
+download the binary for your Operating System.
+
+Place the binary in the root of your project directory.
+
+**Update config file**
+
+Update you collector config with the follwing:
+
+<Tabs>
+<TabItem value="SigNoz Cloud" label="SigNoz Cloud" default>
+
+```yaml
+receivers:
+  # Read more about kafka metrics receiver - https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/kafkametricsreceiver/README.md
+  kafkametrics:
+    brokers:
+      - localhost:9092
+      - localhost:9093
+      - localhost:9094
+    protocol_version: 2.0.0
+    scrapers:
+      - brokers
+      - topics
+      - consumers
+  # Read more about jmx receiver - https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/jmxreceiver/README.md
+  jmx/1:
+    # configure the path where you installed opentelemetry-jmx-metrics jar
+    jar_path: path/to/opentelemetry-jmx-metrics.jar #change this to the path to you opentelemetry-jmx-metrics jar file you downloaded above
+    endpoint: service:jmx:rmi:///jndi/rmi://localhost:2020/jmxrmi
+    target_system: jvm,kafka,kafka-consumer,kafka-producer
+    collection_interval: 10s
+    log_level: info
+    resource_attributes:
+      broker.name: broker1
+  jmx/2:
+    jar_path: ${PWD}/opentelemetry-jmx-metrics.jar
+    endpoint: service:jmx:rmi:///jndi/rmi://localhost:2021/jmxrmi
+    target_system: jvm,kafka,kafka-consumer,kafka-producer
+    collection_interval: 10s
+    log_level: info
+    resource_attributes:
+      broker.name: broker2
+exporters:
+  otlp:
+    endpoint: "ingest.{region}.signoz.cloud:443"
+    tls:
+      insecure: false
+    headers:
+      "signoz-access-token": "<SIGNOZ_INGESTION_KEY>"
+  debug:
+    verbosity: detailed
+
+service:
+  pipelines:
+    metrics:
+      receivers: [kafkametrics, jmx/1, jmx/2]
+      exporters: [otlp]
+```
+
+Depending on the choice of your region for SigNoz cloud, the otlp endpoint will vary according to this table.
```

Comment:
Can you remove the table below and hyperlink to https://signoz.io/docs/ingestion/signoz-cloud/overview/

This doc explains about the endpoints.